packages/nx-cloudflare-wrangler/src/executors/pages/deploy/executor.ts

```diff
@@ -2,14 +2,18 @@ import { ExecutorContext, joinPathFragments } from '@nrwl/devkit';
 import { PagesDeployExecutorSchema } from './schema';
 import { runWranglerCommandForProject } from '../../wrangler';
 import { execSync } from 'child_process';
+import { resolve } from 'path';
 
 export default async function deployExecutor(
     options: PagesDeployExecutorSchema,
     context: ExecutorContext
 ) {
+    if (options.dist) {
+        options.dist = resolve(process.cwd(), options.dist);
+    }
     const dist = joinPathFragments(
         process.cwd(),
-        context.workspace.projects[context.projectName].targets.build.options
+        options.dist ?? context.workspace.projects[context.projectName].targets.build.options
```

Comment:
Is this needed? Options object is spread later in the `deployOptions`, overriding the `dist` you create here.
Maybe it's better to drop this change, that way it's also the same like you did in the serve executor. Or am I missing something?

Reply:
Yes, you're right. Dropped this.